data_diff/databases/presto.py

```diff
@@ -38,7 +37,25 @@ class Presto(Database):
     def __init__(self, **kw):
         prestodb = import_presto()
 
-        self._conn = prestodb.dbapi.connect(**kw)
+        if kw.get("schema"):
+            self.default_schema = kw.get("schema")
+
+        try:
+            # checks if user and password are missing when auth=basic
+            kw.get("auth") == "basic" and "user" in kw and "password" in kw
+            # if auth=basic, add basic authenticator for Presto
+            kw["auth"] = prestodb.auth.BasicAuthentication(kw.pop("user"), kw.pop("password"))
+        except:
```

Comment:
Please catch specific exceptions. I believe this one should be `KeyError` ?

Reply:
Changed it from `ValueError` to `KeyError`

Reply:
You misunderstood. My problem was with your use of `except:`.

Raising ValueError was actually the better raising KeyError.